config/application.rb

```diff
@@ -34,5 +34,7 @@ class Application < Rails::Application
 
     config.action_mailer.default_url_options = { :host => 'localhost' }
     config.user_verification_recipients = ['cgruppioni@law.harvard.edu']
+    config.admin_email = 'cgruppioni@law.harvard.edu'
+    config.cap_api_key = '2c62c54b47e507b2eee20a70f29f1b4ae0ccd1a3'
```

Comment:
Can this be in `development.rb`/`test.rb`, so it doesn't cover up a potential bug on prod/staging?